### PR TITLE
fixed the search bar functionality

### PR DESCRIPTION
**Pull request by Vidhanvyrs**
hey @HimanshuNarware please check the changes and provide me with the feedback

### CODE REVIEW

**Comment by Vidhanvyrs** [comment]
i don't know why it is failing the deployment let me show you the working functionality gonna commit once again
[screencast-localhost_3001-2023.10.04-00_02_37.webm](https://github.com/HimanshuNarware/Devlabs/assets/94836010/44819673-2331-4824-9821-b8c0d48e7fc2)

File: src/Component/Home.js [hunk 1]
```diff
@@ -1,118 +1,105 @@
...
-if(localStorageValue === 'undefined' || localStorageValue==='all'){
-    allvalue=dataBaseData;
-    console.log("No localstorage item");
-}
-else if(localStorageValue!=='all' && localStorageValue!== 'undefined'){
-    console.log(localStorageValue);    
-    if(localStorage.getItem('filter-2')){
-        allvalue=currentPost1.filter((e)=> e.category.toLowerCase().includes(localStorageValue)||e.category.includes(localStorage.getItem('filter-2')));
-    }else{
-        allvalue=currentPost1.filter((e)=> e.category.toLowerCase().includes(localStorageValue));
```
Comment (HimanshuNarware) [inline src/Component/Home.js:41]: why you remove this code, this code  is for search by content type(in the sidebar)

**Comment by Vidhanvyrs** [comment]
because it was having conflict with the main branch and to resolve it i did that thinking it won't change nothing

**Comment by Vidhanvyrs** [comment]
should  I make another PR

**Comment by HimanshuNarware** [comment]
@Vidhanvyrs  you have removed the other code from the file and it's causing lots of errors, don't try to delete the other functionality code,

**Comment by Vidhanvyrs** [comment]
Okay so can i make another PR 

> because it was having conflict with the main branch and to resolve it i did that thinking it won't change nothing
> 
> @Vidhanvyrs sidebar search is not working( this code is for the search by type of tools)

okay i see would try to resolve it now

**Pull request closed without merging**